adapters/repos/db/roaringset/serialization.go

```diff
@@ -154,7 +154,7 @@ func NewSegmentNode(
 	rw.MoveBufferToAbsolutePosition(0)
 	rw.WriteUint64(uint64(offset))
 
-	return &sn, nil
+	return &SegmentNode{contentReader: contentReader.NewMMap(data)}, nil
```

Comment:
This is just used for the CommitLog right? I was wondering why you decided on setting NMap here by default.

Reply:
Ok, reading the code above, I get it know, but I'd advise renaming the `NMap` reader to `Memory` reader or something like that to highlight that it will work with any `[]byte`.